src/test/java/ua/ithillet/lesson5/HillelListImplementTest.java

```diff
@@ -0,0 +1,60 @@
+package ua.ithillet.lesson5;
+
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+public class HillelListImplementTest {
+
+    WorkWithArrays workWithArrays = new WorkWithArrays();
+
+    private static final String[] arrayWords = {"arise", "awake", "be", "bear", "beat", "become", "begin", "bend", "bet",
+            "bind", "bite", "bleed", "blow", "break", "breed", "bring", "build", "buy", "catch", "choose", "cling", "come", "cost",
+            "cut", "deal", "dig", "do", "draw", "drink", "drive", "eat", "fall", "feed", "feel", "fight", "find", "flee", "fly", "forbid",
+            "forget", "forgive", "freeze", "get", "give", "go", "grow", "hang", "have", "hide", "hit", "hold", "hurt", "keep", "know",
+            "lay", "lead", "learn", "leave", "lend", "let", "lie", "light", "lose", "make", "mean", "meet", "pay", "put", "read", "ride"};
+
+    private static final String[] checkArray = {"awake", "become", "bite", "bring"};
+    private static final String[] checkRemoveItem = {"awake", "become", "bring"};
+    private static final String item = "bite";
+
+    private static final String[] checkNegScenarioArray = {"awake", "become", "eat", "bring", "lose"};
+    private static final String itemNeg = "lose";
+
+
+    @Test
+    public void checkPositiveScenarios() {
+        SoftAssertions softAssertions = new SoftAssertions();
+        workWithArrays.add(arrayWords[1]);
```

Comment:
щоб зрозуміти яке тут слово, треба горнути екран вгору і знайти це слово. Пиши просто: 
```suggestion
        workWithArrays.add("awake");
```